components/library/zeromq/zeromq5/zeromq.p5m

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
I'd follow what we've done for gcc versions. And for gcc we just preserved old p5m files for runtime. But I would include dependencies on old zeromq versions to be  respectful to packages dependent on older library/c++/zeromq.